sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/DirectoryAsyncAPITests.groovy

```diff
@@ -541,7 +542,26 @@ class DirectoryAsyncAPITests extends APISpec {
         "fileName"  | -1      | null                                                  | testMetadata                          | ShareErrorCode.OUT_OF_RANGE_INPUT
         "fileName"  | 1024    | new ShareFileHttpHeaders().setContentMd5(new byte[0]) | testMetadata                          | ShareErrorCode.INVALID_HEADER_VALUE
         "fileName"  | 1024    | null                                                  | Collections.singletonMap("", "value") | ShareErrorCode.EMPTY_METADATA_KEY
+    }
+
+    def "Create file lease"() {
+        given:
+        primaryDirectoryAsyncClient.create().block()
+        def leaseId = createLeaseClient(primaryDirectoryAsyncClient.getFileClient("testCreateFile")).acquireLease().block()
```

Comment:
Are all the other tests static names too? I thought we're trying to move away from that to prevent tests from stepping on each other (like one test deletes and then the next test fails because its trying to create it)

Reply:
This is standard in file tests right now. We can use #6933 to track this.